courses/models.py

```diff
@@ -427,6 +428,14 @@ def text_id(self):
         """Gets the courseware_id"""
         return self.courseware_id
 
+    @property
+    def course_number(self):
+        """
+        Returns course_number:
+        i.e. the last part of course.readable_id, after the final +
+        """
```

Comment:
```suggestion
        """
        Returns:
            str: Course number from the readable_id
        """
```

Reply:
"Course number from the readable_id": I guess this needs a little explanation as well, as the course number is fetched from courseware_id which is the last part of readable_id, after the final +.
Anyhow, I have updated the description.